docs/platforms/javascript/guides/nuxt/manual-setup.mdx

```diff
@@ -0,0 +1,148 @@
+
+## Install
+
+```bash {tabTitle:npm}
+npm install @sentry/nuxt --save
+```
+
+```bash {tabTitle:yarn}
+yarn add @sentry/nuxt
+```
+
+```bash {tabTitle:pnpm}
+pnpm add @sentry/nuxt
+```
+
+## Configure
+
+Configuration should happen as early as possible in your application's lifecycle.
+
+To set up the Sentry SDK, register the Sentry Nuxt module and initialize the SDK for client and server. At build time, the Sentry Nuxt Module looks for the following two files:
+
+- Client-Side: `sentry.client.config.ts` in the root containing `Sentry.init`
+- Server-Side: `sentry.server.config.ts` in the root containing `Sentry.init`
+
+In these files, you can specify the full range of <PlatformLink to="/configuration/options">Sentry SDK options</PlatformLink>.
+
+
+### Nuxt Module Setup
+
+Add the Sentry Nuxt Module to your `nuxt.config.ts` file:
+
+```javascript {filename:nuxt.config.ts}
+export default defineNuxtConfig({
+  modules: ["@sentry/nuxt/module"]
+});
+```
+
+Adding this module enables the Sentry SDK in your Nuxt application. The Sentry Nuxt Module will then automatically look for the Sentry configuration files and initialize the SDK accordingly.
+
+### Client-side Setup
+
+Add a `sentry.client.config.ts` file to the root of your project (this is probably the same level as the `package.json`). In this file, import and initialize Sentry, specifying any SDK options for the client:
+
+```javascript {filename:sentry.client.config.ts}
+import * as Sentry from '@sentry/nuxt';
+
+Sentry.init({
+  // If set up, you can use your runtime config here
+  // dsn: useRuntimeConfig().public.sentry.dsn,
+  dsn: "___PUBLIC_DSN___",
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0
+});
+```
+
+### Server-side Setup
+
+1. Add a `sentry.server.config.ts` file to the root of your project:
+
+```javascript {filename:sentry.server.config.ts}
+import * as Sentry from '@sentry/nuxt';
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0
+});
+```
+
+The Nuxt `useRuntimeConfig()` does not work in the Sentry server config due to technical reasons (the config file has to
+be loaded before Nuxt is loaded). To be able to use `process.env` you either have to add `--env-file=.env` to your node command
+
+```bash {tabTitle: node}
+node --env-file=.env .output/server/index.mjs
+```
+
+or use the `dotenv` package:
+
+```javascript {tabTitle: Server Config} {filename:sentry.server.config.ts} {1,3}
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+// ... rest of the file
+```
+
+<Alert level="warning">
+  In the beta state of the Nuxt SDK, some features may not work with certain deployment providers. Check the progress on GitHub: [Compatibility with different Deployment Platforms](https://github.com/getsentry/sentry-javascript/issues/14029)
+</Alert>
```

Comment:
Are we planning on making another docs pass before we go stable? If so, all good! Otherwise, let's remove the "beta" wording. Linking to the issue is fine, though we also might add a [Compatibility section](https://docs.sentry.io/platforms/javascript/guides/sveltekit/#compatibility) to the getting started page when we rework it to reference the wizard.

Reply:
I would still leave this in for now. Will add the compatibility section 👍